src/viking.js

```diff
@@ -1,14 +1,56 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength
+  };
+
+  receiveDamage(damage) {
+    this.health -= damage
+  };
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (Viking === "alive"){
```

Comment:
Viking inherited "this.health" from Soldier. To see if the Viking is still alive you need to check the health of the Viking. If the number of health is lower or equal 0, the Viking died. If the health is bigger than 0, the Viking is still alive. In code:

```
  receiveDamage(damage) {
    this.health -= damage;
    if (this.health > 0){
      return (`${this.name} has received ${damage} points of damage`);
    } else {
      return (`${this.name} has died in act of combat`)
    }
  };

```